docs/configure.md

```diff
@@ -312,7 +312,7 @@ Increasing the number of `gitserver` replicas can improve performance when your
 To change the number of `gitserver` replicas:
 
 1. Update the `replicas` field in [gitserver.StatefulSet.yaml](../base/gitserver/gitserver.StatefulSet.yaml).
-1. Update the `SRC_GIT_SERVERS` environment variable in all services to reflect the number of replicas.
+1. Update the `SRC_GIT_SERVERS` environment variable in the frontend service to reflect the number of replicas.
```

Comment:
Please update the convenience script below (L353)

Reply:
I looked into this just now.

(1) It does still work, but is more complex than we need in the long run.
(2) We cannot simplify it right now because the xlang-go and xlang-go-bg deployments still need to have gitservers updated (but this is changing soon as those are removed). We can simplify it once those are removed.